file-mon/src/main/java/com/wipro/ats/bdre/filemon/FileScan.java

```diff
@@ -52,7 +54,7 @@ public static void scanAndAddToQueue() {
                 fileCopyInfo = new FileCopyInfo();
                 fileCopyInfo.setFileName(fileName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Call the static method Integer.toString(...) instead of instantiating a temporary object to perform this to string conversion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1158)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "Integer.toString" instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2131)